samples/performance/subpasses/subpasses_tutorial.md

```diff
@@ -54,59 +54,58 @@ vec4 world_w = inv_view_proj * clip;
+![Subpasses vs render passes trace](images/subpasses-renderpasses-trace.jpg)
 
 ## Merging
 
 As stated by the Vulkan reference, _Subpasses with simple framebuffer-space dependencies may be merged into a single tile rendering pass, keeping the attachment data on-chip for the duration of a renderpass._ [[2](#references)].
 
-Since sub-passes information is known ahead of time, the driver is able to detect if two or more subpasses can be merged together. The consequence of this is that [vkCmdNextSubpass](https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkCmdNextSubpass.html) becomes a `NOP`.
+Since subpass information is known ahead of time, the driver is able to detect if two or more subpasses can be merged together. The consequence of this is that [vkCmdNextSubpass](https://www.khronos.org/registry/vulkan/specs/1.1-extensions/man/html/vkCmdNextSubpass.html) becomes a `NOP`.
 
 In other words, a GPU driver can optimize even more by merging two or more subpasses together as long as certain requirements are met. Such requirements may vary between vendors, the following are the ones for Arm GPUs:
 
-* If color attachment data formats can be merged.
 * If merge can save a write-out/read-back; two unrelated subpasses which don't share any data do not benefit from multipass and will not be merged.
 * If the number of unique `VkAttachments` used for input and color attachments in all considered subpasses is <= 8. Note that depth/stencil does not count towards this limit.
 * The depth/stencil attachment does not change between subpasses.
 * Multisample counts are the same for all attachments.
 
-Furthermore, in order to be merged, sub-passes are required to use at most 128-bit per pixel of *tile buffer color storage*, although some of the more recent GPUs such as Mali-G72 increase this to up to 256-bits per pixel. G-buffers requiring more color storage than this can be used at the expense of needing smaller tiles during fragment shading, which can reduce overall throughput and increase bandwidth reading tile lists.
-
-From the sample perspective, the best way to verify whether two subpasses are merged or not is to compare the physical tiles (`PTILES`) counter by switching between the sub-passes and the render passes technique. Two fused sub-passes will need half the number of `PTILES` needed by two render passes, indeed comparing the following screenshot with the previous one, roughly half the number of tiles are used (`0.5` vs `1.1`) and about `70%` of bandwidth is saved.
+Furthermore, in order to be merged, subpasses are required to use at most 128-bit per pixel of *tile buffer color storage*, although some of the more recent GPUs such as Mali-G72 increase this to up to 256-bits per pixel. G-buffers requiring more color storage than this can be used at the expense of needing smaller tiles during fragment shading, which can reduce overall throughput and increase bandwidth reading tile lists.
```

Comment:
I think this sentence could use a tweak to clarify that the limits are Mali specific. Something like:

> On Mali GPUs, subpass merging is limited to subpasses which utilize at most 128-bits per pixel of *tile buffer color storage*, although some of the more recent Mali GPUs such as Mali-G72 relax this limit to 256-bits per pixel.